spec/system/task_spec.rb

```diff
@@ -85,8 +81,7 @@
 
       it '既にステータスが完了のタスクのステータスを変更した場合、Taskの完了日が更新されないこと' do
         # TODO: FactoryBotのtraitを利用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id, status: :done, completion_date: Time.current.yesterday)
+        task = FactoryBot.create(:task, :done, project_id: project.id)
```

Comment:
`FactoryBot.`の部分は設定ファイルに追記を行って省略した方がコード量が減るので良いですね。
```suggestion
        task = create(:task, :done, project_id: project.id)
```

Reply:
修正しました。